.nvmrc

```diff
@@ -1 +1 @@
-v16.13.0
+v18.12.0
```

Comment:
Let's use Node LTS

Reply:
updated to 18.20.3

Reply:
@omerzahidbajwa currently Node LTS is 20.15.0

Reply:
Thanks for pointing out, i was using the long term support version for v18 before. Updated to v20.15.0